docs/elements/hx-drawer/index.html

```diff
@@ -0,0 +1,113 @@
+---
+title: <hx-drawer>
+also:
+  components/drawer: Drawer
+---
+{% extends 'element.njk' %}
+{% import '_utils.njk' as utils %}
+{% block page_header %}
+  <p>
+    The <code>{{page.title}}</code> is a container for a drawer dialog,
+    anchored to the right-side of the screen.  It is used to build a
+    <a href="components/drawer">Drawer</a> component.
+  </p>
+{% endblock %}
+
+{% block content %}
+  <section>
+    <dl class="metadata hxList">
+      <div>
+        <dt>Permitted Parents</dt>
+        <dd>any</dd>
+      </div>
+      <div>
+        <dt>Permitted Children</dt>
+        <dd>
+          In this order:
+          <ol class="hxList">
+            <li>
+              {{ utils.element('header') }}
+            </li>
+            <li>
+              {{ utils.customElement('hx-alert') }}
+              <small>(optional)</small>
+            </li>
+            <li>
+              {{ utils.customElement('hx-div') }}
+            </li>
+            <li>
+              {{ utils.element('footer') }}
```

Comment:
removed "(optional)" text from footer child. @araiford said that it's required, for now